modules/programs/convert_img/convert_img.py

```
@@ -139,6 +141,144 @@ def parse_args() -> argparse.Namespace:
+        "webp": "WEBP",
+        "tiff": "TIFF",
+        "bmp": "BMP",
+        "gif": "GIF",
+        "avif": "AVIF",
+        "heif": "HEIF",
+        "ico": "ICO",
+    }
+    return format_map.get(format_name, "PNG")
+
+
+def prepare_image_for_format(img: Image.Image, output_format: str) -> Image.Image:
```

Flake8 (7.2.0)

[error] line too long (82 > 79 characters)

(E501)